### ISSUE TITLE
Not able to build the project with this module added in dependencies

### ISSUE THREAD

**Issue by Mubashirkhan** [mauron85/react-native-background-geolocation#145]
This is the error while building the project i . have added my build.gradle file as attachment 
Need Help with this issue
```
/android/app/src/main/AndroidManifest.xml:27:9-31 Error:
        Attribute meta-data#android.support.VERSION@value value=(25.3.1) from [com.android.support:design:25.3.1] AndroidManifest.xml:27:9-31
        is also present at [com.android.support:support-v4:26.0.0-alpha1] AndroidManifest.xml:27:9-38 value=(26.0.0-alpha1).
        Suggestion: add 'tools:replace="android:value"' to <meta-data> element at AndroidManifest.xml:25:5-27:34 to override.

See http://g.co/androidstudio/manifest-merger for more information about the manifest merger.

:app:processDebugManifest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':app:processDebugManifest'.
> Manifest merger failed : Attribute meta-data#android.support.VERSION@value value=(25.3.1) from [com.android.support:design:25.3.1] AndroidManifest.xml:27:9-31
        is also present at [com.android.support:support-v4:26.0.0-alpha1] AndroidManifest.xml:27:9-38 value=(26.0.0-alpha1).
        Suggestion: add 'tools:replace="android:value"' to <meta-data> element at AndroidManifest.xml:25:5-27:34 to override.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
```

> dependencies {
    compile project(':react-native-mauron85-background-geolocation')
    compile project(':react-native-wheel-picker')
    compile project(':react-native-share')
    compile project(':react-native-keep-awake')
    compile project(':react-native-push-notification')
    compile project(':react-native-cookie')
    compile project(':react-native-twitter')
    compile project(':react-native-fbsdk')
    compile project(':react-native-twitter-signin')
    compile project(':realm')
    compile project(':react-native-blur')
    compile project(':react-native-config')
    compile project(':react-native-image-picker')
    compile project(':react-native-vector-icons')
    compile project(':react-native-navigation')
    compile project(':react-native-device-info')
    compile(project(':react-native-google-places')){
        exclude group: 'com.google.android.gms', module: 'play-services-base'
        exclude group: 'com.google.android.gms', module: 'play-services-places'
        exclude group: 'com.google.android.gms', module: 'play-services-location'
    }
    compile fileTree(dir: "libs", include: ["*.jar"])
    compile 'com.android.support:appcompat-v7:24.2.1'
    compile('com.facebook.android:facebook-android-sdk:4.26.0')
    compile "com.facebook.react:react-native:+"  // From node_modules
    compile "com.google.android.gms:play-services-base:11.+"
    compile 'com.google.android.gms:play-services-places:11.+'
    compile 'com.google.android.gms:play-services-location:11.+'
    compile 'com.google.android.gms:play-services-gcm:11.+'
}

[build.gradle.txt](https://github.com/mauron85/react-native-background-geolocation/files/1606845/build.gradle.txt)

**Comment by AndrewTolo** [comment]
+ 1  Have same error after added :react-native-mauron85-background-geolocation

**Comment by gusilveiramp** [comment]
same here...

**Comment by jaseemabbas** [comment]
+1

**Comment by mauron85** [comment]
There is conflict in android support library within multiple react modules. There is not much I can do about it. You can try to edit file

https://github.com/mauron85/react-native-background-geolocation/blob/master/android/lib/build.gradle

and replace line: `compile 'com.android.support:support-v4:+'`
with version used by conflicting react module (you need to figure which one is it by yourself).

**Comment by mauron85** [comment]
Also I would say you dependencies from build.gradle you've posted are in direct conflict:

eg:
```
compile 'com.android.support:appcompat-v7:24.2.1'
```

**Comment by mauron85** [comment]
I guess this was fixed in 0.5 branch. Reopen if problem still persists.
